packages/block-editor/src/components/list-view/style.scss

```diff
@@ -207,7 +207,7 @@
 	}
 
 	.block-editor-list-view-block__menu-cell {
-		padding-right: 5px;
+		padding-right: 4px;
```

Comment:
Reduced this right padding by `1px` so that numerically each of the different widths of the components in the row don't exceed the default `350px` for the panel (without this change, the panel winds up being `351px`). `4px` seems a bit more consistent with other spacing values in components, and also is a bit neater than forcing the max-width of the label wrapper to be `239px`. Happy to move this around if anyone feels strongly about it.

Reply:
+1. We can even use `$grid-unit-05`.

Reply:
Oh, good call, I'll switch this over to that variable 👍